src/OptimisticTokenVotingPlugin.sol

```diff
@@ -203,6 +209,8 @@ contract OptimisticTokenVotingPlugin is
     }
 
     /// @notice Determines whether L2 votes are currently usable for voting
+    // Q(daniel): not sure if we should use the last L2 block or the last verified L2 block...
```

Comment:
If we should, then we will :)

Reply:
I think we should, but this can be done later. I still need to merge my PR first.

Reply:
Which PR, what changes do you mean?

Reply:
https://github.com/taikoxyz/taiko-mono/pull/17868 This one, which is part of the main branch, not in a release branch.